portfolio/src/main/webapp/index.html

```diff
@@ -103,7 +103,14 @@ <h2>Comments</h2>
              <input type="text" name="user-comment" size = "50">
              <br>
              <br>
+             <label for="max-comments">Show Comments: (between 1 and 5):</label>
+             <input type="number" id="max-comments" min="1" max="5" onchange="getPhrase()" value = "2">
```

Comment:
nit: might want to make the spacing for value consistent with the rest.

ie: `value="2"`